cmd/upgrade.go

```diff
@@ -169,6 +192,25 @@ func (d *diffCmd) runHelm3() error {
 		return fmt.Errorf("Failed to render chart: %s", err)
 	}
 
+	if d.threeWayMerge {
```

Comment:
Probably L171-L193 doesn't need to be run when we enter the 3-way merge mode?

Reply:
Nevermind. It turned out correct as we still render the chart at L190 to be used when building K8s objects at #L207.